### PR TITLE
Best of both worlds: hyde + zen = hybrid

### PR DESCRIPTION
**Pull request by encarsia**
Hello there,

after having fiddled around with the zen themes but also liking the clean looks of hyde I am finally using a mixture of both as a hyde variant for [my personal site](https://encarsia.github.io/):

* From the zen(-forkawesome) theme it features ForkAwesome font icons, fancydates and post header information (date/author/translations/comment status).
* Subthemes: hyde color or custom schemes are supported.
* Sidebar options are extended:
  * navigation links can be text only (hyde) or icon only (zen) or combined
  * turn blog title and description in navigation bar on (hyde) or off (zen)

So I hope this meets your requirements for a theme variant, details on usage are provided in the README and conf.py.sample.

An open question for me:
As I am using Nikola v8 I put this theme in the corresponding theme folder. The hyde theme is located in the v7 folder but it works normal with v8. What are the criteria for a theme to be stored here or there?

Regards,

### CODE REVIEW

**Comment by Kwpolska** [comment]
Themes in the `v7/` directory might not be fully compatible with v8 and they might need porting, mainly with math, baguetteBox, and other scripts.

File: v8/hybrid/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+You can read a tutorial about how to create/port new theme [at Nikola's site](https://getnikola.com/creating-a-theme.html)
+
+(*Note:* The `.sidebar` class was renamed to `.hsidebar` to avoid conflicts with reST.)
+
+License is MIT
+
+Known Issues:
+
+* Not all features of Nikola are tested.
+* Galleries will work better when [Issue #1764](https://github.com/getnikola/nikola/issues/1764) is fixed.
```
Comment (Kwpolska) [inline v8/hybrid/README.md]: It’s closed. The scripts we use for galleries are available in `base` and you can find examples in the built-in themes on how to use them.

File: v8/hybrid/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+The Hybrid theme is based on the Hyde theme which is a port of the [Hyde theme by mdo](http://hyde.getpoole.com/) combined with features from the Zen theme family:
+
+* ForkAwesome font icons are supported
+* fancydates JS plugin is used
+* navigation links can be text only or icon only or combined
+* turn blog title and description in navigation bar on/off
+* Hyde color schemes are supported
+
+Specify preferences via GLOBAL_CONTEXT variable in the config:
```
Comment (Kwpolska) [inline v8/hybrid/README.md]: You might want to port this to v8’s `THEME_CONFIG` mechanism.

File: v8/hybrid/README.md (hunk 1 above)
Comment (encarsia) [reply v8/hybrid/README.md]: This was just plain copied from the hyde README so I removed these lines in both.

File: v8/hybrid/README.md (hunk 2 above)
Comment (encarsia) [reply v8/hybrid/README.md]: Done.

**Comment by encarsia** [comment]
Is there anything else I can do here?

**Comment by Kwpolska** [comment]
I slightly modernized `hyde` for v8, added gallery support to both `hyde` and `hybrid`, and published the themes to `themes.getnikola.com`. Thanks!

**Comment by encarsia** [comment]
Thanks, much appreciated.

**Pull request merged**